scripts/install_ilm_policy.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+from __future__ import absolute_import
+from builtins import open
+from future import standard_library
+standard_library.install_aliases()
+
+import argparse
+import json
+
+from elasticsearch.client import IlmClient
+from hysds.es_util import get_mozart_es
+
+
+mozart_es = get_mozart_es()
+
+
+def get_parser():
+    parser = argparse.ArgumentParser(description="Tool to install Mozart ES ILM Policy")
+    parser.add_argument(
+        "--policy_file",
+        required=True,
+        type=str,
+        help="Specify the Mozart ILM policy file.",
+    )
+    return parser
+
+
+if __name__ == "__main__":
+    args = get_parser().parse_args()
+
+    with open(args.policy_file) as f:
+        ilm_policy = json.load(f)
+
+    # https://elasticsearch-py.readthedocs.io/en/7.x/api.html#elasticsearch.client.IlmClient
+    # ignore 400 cause by IndexAlreadyExistsException when creating an index
+    es_ilm = IlmClient(mozart_es.es)
+    es_ilm.put_lifecycle(policy="ilm_policy_mozart", body=ilm_policy)
```

Comment:
i'd recommend adding a new parameter for the policy name and have it default to `ilm_policy_mozart`

Reply:
Good idea. It's done:

![image](https://github.com/hysds/mozart/assets/42812746/dfdfa8ee-51d1-4fc1-b6a9-33c6e9e66360)